### ISSUE TITLE
Create home register screen for UgandaHPV

### ISSUE THREAD

**Issue by cagulas** [OpenSRP/opensrp-client-uganda-hpv#8]
<img width="442" alt="screen shot 2018-03-07 at 12 43 29 pm" src="https://user-images.githubusercontent.com/3488986/37085109-44800b42-2205-11e8-87c0-87f495c4f567.png">

- [x] The home register screen is what you land on when you first log in to the app. Records are sorted by last modified (same as ZEIR).
- [x] The Name column shows the girl's first and last name, caretaker name, and age (`first_name` `family_name`, `caretaker_name`, and `age`)
- [x] The ID column will just show one ID number `opensrp_id`
- [x] The Dose (D) column will show which HPV dose is due, upcoming, or complete. Please refer to the schedule for this (https://github.com/OpenSRP/opensrp-client-uganda-hpv/issues/5).
- [x] Tapping on a red or blue reminder opens the vaccine widget for that dose. The girl's profile view also opens in the background (same as ZEIR).
- [x] When both HPV doses are given, the Complete status shows instead.
- [x] For Dose 2, show the date that Dose 1 was given underneath
- [x] For Dose 2, if it's not due yet (i.e. upcoming) it is grey and not tappable.
- [x] Search bar allows the user to search by `first_name` `family_name` or `opensrp_id`
- [x] Not pictured in the mockup, but let's show the Scan QR code button.
- [x] LG are the user's initials.
- [x] The location dropdown shows the user's health facility location by default. The other dropdown choices will be the names of the schools that are assigned to the health facility location in the location hierarchy. Whichever location is selected will be the encounter location in OpenMRS.
- [x] The plus button opens the Registration form (https://github.com/OpenSRP/opensrp-client-uganda-hpv/issues/1)
- [x] The three dots button on the top right opens a small menu with log out and change language options (same as TBR).

**Comment by cagulas** [comment]
Results from QA of 11 April release:

- [ ] Let's make the white circle and the BD initials smaller, more along the lines of what you see in the mockup.
- [ ] Let's remove the "s" from "names" in the search bar.
- [ ] When you search by QR code, it's not obvious how to cancel the search, because the ID number isn't displayed in the search bar. Can we show the ID from the QR code scan in the search bar so it's more obvious you press the X to clear it?
- [ ] The X button in the search bar is too right-centered. Can we add some space to the right of it, so it's not touching the end of the search bar?
- [ ] When I registered the first girl, she immediately appeared, but when I registered the next two girls, they didn't appear in the register list until I logged out and logged back in. They should appear immediately in the list at the top after registration.
- [ ] Column heading "Dose (D)" doesn't look centered in the column.
- [ ] The app crashes when I click on girls "Rachel Henry" and "Rebecca Harry" in the register view each time. I can only open the record for the first girl I registered, Carol Gulash.
